src/Catalog/Helpers/FeedPackageDetails.cs

```diff
@@ -25,14 +26,16 @@ public sealed class FeedPackageDetails
             DateTime lastEditedDate,
             DateTime publishedDate,
             string packageId,
-            string packageVersion)
+            string packageVersion,
```

Comment:
Maybe we should rename this to `normalizedVersion`? Same comment to line 15

Reply:
I worry that doing this rename will lead to a number of changes all over the project. I will check it out and see how much is impacted. If it generates too many changes, I think adding an XML doc that explains that it's the normalized version would be sufficient.

Reply:
Didn't effect nearly as much as I expected. Will do.